kubernetes/stable.yml

```diff
@@ -49,33 +56,40 @@ kubernetes:
   - name: checkout-io-tests-cronjob
     kind: cronjob
     schedule: "* * * * *"
-    activeDeadlineSeconds: 2400
+    activeDeadlineSeconds: 10800 # 3 hours
```

Comment:
I think the issue was that I increased this time enough that all of the tests are currently being executed, and the ones that are failing on saturday wee hours weren't before :-/

Reply:
@wdsrocha what do you think? should we decrease this time to match what we have running right now and later try to fix the test or should we keep it running _everything_ and not alert us when there are issues?